guides/common/modules/proc_promoting-a-content-view.adoc

```diff
@@ -51,5 +51,24 @@ Now the repository for the Content View appears in all environments.
 ----
 +
 Now the database content is available in all environments.
-
++
 To register a host to your Content View, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[Registering Hosts] in _{ManagingHostsDocTitle}_.
++
+. Promote Content Views across all lifecycle environments within an organization using the following Bash script:
++
+[source, Bash, options="nowrap" subs="verbatim,quotes"]
+----
+ORG="_My_Organization_"
+CVV_ID=_3_
```

Comment:
```suggestion
CVV_ID=_My_Content_View_ID_
```

same below in line 73. Is this a composite content view or content view? I'm asking because of the variable name. Looking at the text, it's a Content View, so maybe rename the variable to `CV`?

Reply:
CVV is content view version. The script uses `hammer content-view version promote`, so I think CVV_ID is correct.